### DISCUSSION TITLE
Recommended ESP board that is reliable and available on US Amazon site?

### DISCUSSION THREAD

**Discussion by sean-leach** [gysmo38/mitsubishi2MQTT#276] [Q&A]
I have five mini splits running mitsubishi2MQTT, and it's aweosme. Sadly two of the units seem to keep killing the ESP boards in them. Can anyone recommend a solid ESP board I can get on the US Amazon site that has been reliable, doesn't require resistors to be wired in, etc?

Thanks!

**Comment by lcoppenr** [comment]
I use the FLIP_C3 boards without level shifters and haven't had any issues yet.   https://www.amazon.com/VDBX-io-FLIP_C3-ESP32-TASMOTA-Compatible/dp/B0CKWSTFP8/ref=sr_1_1?crid=2DB05A3ZTNXSH&dib=eyJ2IjoiMSJ9.iF-oR6FHx2RhycVfjWgaNTOt5_mmDHxDuRciT6y-9tNvPGSVpgwQsShl-_yA4r40OEkq7rSN4frLOuRMVXoFGa6liP2JNPbn7jXrUwgVarxs9ctT8dlEx93wHL4oL17sG14hX4lJYEb1Z6rSplK6mBJubQJ-9EryamvZZeSXHwbf2KeE9b0OWYGAGRR1mMPpY2HajkqpkA7QkZ0Vb0stTjfMzOHIrFw-Eof-v2SkAalGbOqiR3yrPTjctGMQ-MHTAz1YYdgSR9K-ktNvsZEPaK40-epn4sU46SZO2Zzl5K8.bfQOmR8RgeOQrhjOS1XjNGKDTdySVTqC7r4trkwEZVw&dib_tag=se&keywords=flip_c3&qid=1726596207&sprefix=flip_c3%2Caps%2C144&sr=8-1
